src/config.js

```diff
@@ -127,6 +127,14 @@ const parser = yargs
                 default: false,
                 group: 'Proxy options',
             },
+            'proxy-forward-user-attr': {
+                describe:
+                    'Forwarded attributes of the authenticated user to the upstream server.',
+                type: 'string',
+                default: [],
+                group: 'Proxy options',
+                array: true,
```

Comment:
Given that we're passing in a json object does this need to be an array? Would it make more sense to only pass one object with the whole mapping?

Reply:
I'm not sure. The idea was making it a little more legible, and easy to break lines ex.

This example show how it would looks like if passing a single json, I know it works as well, but the value is a little more polluted and breaks to next line.
```
yarn start --proxy-host=upstream-service:9000 \
    --proxy-forward-user-attr={"X-Forwarded-Login":"email", "X-Forwarded-Email":"email", "X-Forwarded-Name":"email"}
```

This example it's more verbose but allows break lines more easily, and it's more readable (in my opinion).
```
yarn start --proxy-host=upstream-service:9000 \
    --proxy-forward-user-attr={"X-Forwarded-Login":"email"} \
    --proxy-forward-user-attr={"X-Forwarded-Email":"email"} \
    --proxy-forward-user-attr={"X-Forwarded-Name":"email"}
```

Reply:
Also, Initially I was trying to think a simpler way to define key => value notation, but I though `json` would be more intuitive.

Another option would be something like:

```
yarn start --proxy-host=upstream-service:9000 \
    --proxy-forward-user-attr=X-Forwarded-Login:email \
    --proxy-forward-user-attr=X-Forwarded-Email:email \
    --proxy-forward-user-attr=X-Forwarded-Name:email
```